hostports/watcher.go

```diff
@@ -109,6 +111,25 @@ func (p PortRule) natIptables() []byte {
 	return buf.Bytes()
 }
 
+func (p ChainRule) filterIptables() []byte {
```

Comment:
At first, I wanted to use `network_mode` label to distinguish the scenarios.
But you don't want to do like that.
I can only figure it out indirectly.